ui/StatusQ/src/StatusQ/Core/StatusIcon.qml

```diff
@@ -17,7 +17,7 @@ ColorImage {
             source = icon
             objectName = "custom-icon"
         } else if (icon !== "") {
-            source = "../../assets/img/icons/" + icon+ ".svg";
+            source = Qt.resolvedUrl("../../assets/img/icons/" + icon+ ".svg");
```

Comment:
`StatusEmoji` ,`CurrenciesModel` and maybe other components probably need the same fix

Reply:
Will create separate ticket to investigate it.

Reply:
https://github.com/status-im/status-desktop/issues/17451